Makefile

```diff
@@ -34,12 +34,17 @@ $(info $(shell printf "%-21s = %-10s\n" "KUBE_CONTROLLERS_VER" $(KUBE_CONTROLLER
 $(info $(shell printf "%-21s = %-10s\n" "TYPHA_VER" $(TYPHA_VER)))
 ##############################################################################
 
-serve:
-	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
+serve: helm
+	# We have to override JEKYLL_DOCKER_TAG which is usually set to 'pages'. 
+	# When set to 'pages', jekyll starts in safe mode which means it will not
+	# load any plugins. Since we're no longer running in github-pages, but would
+	# like to use a docker image that comes preloaded with all the github-pages plugins,
+	# its ok to override this variable.
+	docker run -e JEKYLL_DOCKER_TAG="" -v $$PWD/bin/helm:/usr/local/bin/helm:ro --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
 
 .PHONY: build
-_site build:
-	docker run --rm -ti -e JEKYLL_UID=`id -u` -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll build --incremental $(CONFIG)
+_site build: helm
+	docker run -e JEKYLL_DOCKER_TAG="" -v $$PWD/bin/helm:/usr/local/bin/helm:ro --rm -ti -e JEKYLL_UID=`id -u` -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll build --incremental $(CONFIG)
```

Comment:
command is getting kind of long - might want to split onto multiple lines.

Also, looks like it is duplicated above in the `serve` target almost exactly, so maybe a good idea to pull out into a variable. Just a suggestion though, not required. 

e.g.
- `$(JEKYLL_RUN) jekyll build`
- `$(JEKYLL_RUN) jekyll serve`

Reply:
I ended up just splitting it over multiple lines for clarity. The `serve` step needs to pass a docker arg `-p 4000:4000`, whereas the build command does not. I didn't feel like sorting through that right now...